test/__snapshots__/index.test.js.snap

```diff
@@ -22,8 +22,8 @@ Build \\"aliasMapping\\" to dist:
 46 B: alias-mapping.js.br
 62 B: alias-mapping.esm.js.gz
 46 B: alias-mapping.esm.js.br
-118 B: alias-mapping.umd.js.gz
```

Comment:
I've updated the tests now that #495 is merged, so only two should be failing. We can actually skip updating the snapshots for that in this PR since it's a real bug still to be fixed elsewhere :)